freqtrade/freqai/data_drawer.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, full_path: Path, config: Config, follow_mode: bool = False):
                 "model_filename": "", "trained_timestamp": 0,
                 "data_path": "", "extras": {}}
         self.metric_tracker: Dict[str, Dict[str, Dict[str, list]]] = {}
+        self.current_candle: datetime = datetime.fromtimestamp(637887600)
```

Comment:
you'll want to be careful here - `freqai_interface.py` set's the date with timezone (so does the dataframe) - so assuming this is never reset/set initially for whatever reason, you'd get a comparison error (can't compare timezones with and without timezone).

i'm also not sure why we're using a default of March 1990 here - in other places we use 0 (1970) for default dates - which works just as well and isn't some magic constant number, but something people can relate to without first explicitly converting the number.

Reply:
I handled the timezone change. 

We are using an arbitrary initialization date, I added a comment to clarify.